### ISSUE TITLE
top memory usage is over teh top after last update

### ISSUE THREAD

**Issue by ayghor** [voidlinux/void-packages#7998]
idk if its related to teh update itself, but i just ran `xbps-install -Su` and rebooted and then i noticed that `top` is over nine thousaaand:

```
top - 23:55:47 up 19 min,  1 user,  load average: 0,68, 0,98, 0,86
Tasks: 171 total,   1 running, 170 sleeping,   0 stopped,   0 zombie
%Cpu(s):  13,6/3,3    17[|||||||||||||||||                                                                                   ]
KiB Mem :112599035281408,0/16382684 [||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
KiB Swap:  0,0/33554428 [                                                                                                    ]

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
 1105 rogi      20   0 5379764 3,303g 169992 S  59,5 21,1  12:17.73 Web Content
 1053 rogi      20   0 2829888 820608 157644 S   6,0  5,0   2:39.96 firefox
  973 root      20   0  358388  79520  67568 S   1,7  0,5   0:14.41 Xorg
 1039 rogi      20   0   60428  12908   6888 S   0,7  0,1   0:00.53 st
   41 root      39  19       0      0      0 S   0,3  0,0   0:00.69 khugepaged
  767 root     -51   0       0      0      0 S   0,3  0,0   0:08.67 irq/31-iwlwifi
 1015 rogi      20   0   52280   7272   6460 S   0,3  0,0   0:00.35 dwm
14574 rogi      20   0   15892   2816   2504 R   0,3  0,0   0:01.28 top
    1 root      20   0     944      4      0 S   0,0  0,0   0:00.95 runit
    2 root      20   0       0      0      0 S   0,0  0,0   0:00.00 kthreadd
    4 root       0 -20       0      0      0 S   0,0  0,0   0:00.00 kworker/0:0H
    6 root       0 -20       0      0      0 S   0,0  0,0   0:00.00 mm_percpu_wq
    7 root      20   0       0      0      0 S   0,0  0,0   0:00.01 ksoftirqd/0
    8 root      20   0       0      0      0 S   0,0  0,0   0:00.24 rcu_preempt
    9 root      20   0       0      0      0 S   0,0  0,0   0:00.00 rcu_sched
   10 root      20   0       0      0      0 S   0,0  0,0   0:00.00 rcu_bh
   11 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 migration/0
   12 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 watchdog/0
   13 root      20   0       0      0      0 S   0,0  0,0   0:00.00 cpuhp/0
   14 root      20   0       0      0      0 S   0,0  0,0   0:00.00 cpuhp/1
   15 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 watchdog/1
   16 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 migration/1
   17 root      20   0       0      0      0 S   0,0  0,0   0:00.00 ksoftirqd/1
   19 root       0 -20       0      0      0 S   0,0  0,0   0:00.00 kworker/1:0H
   20 root      20   0       0      0      0 S   0,0  0,0   0:00.00 cpuhp/2
   21 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 watchdog/2
   22 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 migration/2
   23 root      20   0       0      0      0 S   0,0  0,0   0:00.00 ksoftirqd/2
   24 root      20   0       0      0      0 S   0,0  0,0   0:00.01 kworker/2:0
   25 root       0 -20       0      0      0 S   0,0  0,0   0:00.00 kworker/2:0H
   26 root      20   0       0      0      0 S   0,0  0,0   0:00.00 cpuhp/3
   27 root      rt   0       0      0      0 S   0,0  0,0   0:00.00 watchdog/3
```

it was ok before. `free -h` output is ok:

```
$ free -h
              total        used        free      shared  buff/cache   available
Mem:            15G        4,1G         10G        133M        1,0G         12G
Swap:           31G          0B         31G
```

**Comment by Vaelatern** [comment]
Can't replicate.

**Comment by ayghor** [comment]
i wasnt updating in a whiel. seems that something changed triggering this:

https://gitlab.com/procps-ng/procps/issues/64

anyway it should be fixed on 3.3.13.
<3
